server/config/default.json

```
@@ -6,10 +6,12 @@
     "uri": "mongodb://127.0.0.1:27017/flux-retour-cfas?retryWrites=true&w=majority"
   },
   "apiKeys": {
-    "ymag": "12345"
+    "ymag": "12345",
+    "gesti": "4567"
   },
   "users": {
-    "ymag": "ymag",
+    "ymag": {"name": "ymag"},
+    "gesti": {"name": "gesti"},
```

Vu que tu as mis un attribut name ici on peut revoir la structure et : 

- Virer les "apiKeys"
- Mettre un truc du genre : 

```suggestion
    "ymag": {"name": "ymag", "password": "mdpYmag", "apiKey": "12345"},
    "gesti": {"name": "gesti", "password": "mdpGesti", "apiKey": "12345"},
```

Et refacto le code pour s'intégrer à ca.
Qu'en penses tu ?

---

Yes parfait, j'ai été un peu frileux sur les changements de config !